tests/test_config_consistency.py

```diff
@@ -555,19 +557,59 @@ def test_log_injection_128bit_traceid_disabled(self):
 @scenarios.runtime_metrics_enabled
 @features.tracing_configuration_consistency
 class Test_Config_RuntimeMetrics_Enabled:
-    """Verify runtime metrics are enabled when DD_RUNTIME_METRICS_ENABLED=true"""
+    """Verify runtime metrics are enabled when DD_RUNTIME_METRICS_ENABLED=true and that they have the proper tags"""
 
-    # This test verifies runtime metrics by asserting the prescene of a metric in the dogstatsd endpoint
-    def test_config_runtimemetrics_enabled(self):
-        for data in interfaces.library.get_data("/dogstatsd/v2/proxy"):
-            lines = data["request"]["content"].split("\n")
-            metric_found = False
-            for line in lines:
-                if runtime_metrics[context.library.library] in line:
-                    metric_found = True
-                    break
-            assert metric_found, f"The metric {runtime_metrics[context.library.library]} was not found in any line"
-            break
+    def setup_main(self):
+        self.req = weblog.get("/")
+
+    def test_main(self):
+        assert self.req.status_code == 200
+
+        time.sleep(12)
+        runtime_metrics = [
+            metric
+            for _, metric in interfaces.agent.get_metrics()
+            if metric["metric"].startswith("runtime.") or metric["metric"].startswith("jvm.")
+        ]
+        assert len(runtime_metrics) > 0
```

Comment:
lol I had I was thinking if we could assert the count of expected metrics in case something breaks one, then found this, but not sure how much that can change per lang, timing, env or other factor so may be hard to test.

Reply:
Yeah, maybe we can improve it later? Just testing the existence seems like a good first step 😅